server/src/handlers/http/middleware.rs

```diff
@@ -164,6 +171,31 @@ where
         /* ## Section end */
 
         let auth_result: Result<_, Error> = (self.auth_method)(&mut req, self.action);
+        let body = json!([
```

Comment:
I will work on the Schema, will get back to you in a week's time

Reply:
Hey @de-sh, just following up. Did you get a chance to work on this?

Reply:
Hey, sorry that this got lost, here is what I have been able to come up with:
```json
{
  "audit_id": "String: uuid",
  "timestamp": "String: Datetime",
  "actor": {
    "type": "Enum: user/sync",
    "user_id": "String: username",
    "authorization_method": "Enum: basic/oauth/jwt"
  },
  "request": {
    "method": "Enum: POST/GET/PUT",
    "url": "String: url",
    "headers": {
      "key": "value"
    },
    "body": "String: json",
    "protocol": "Enum: kafka/http"
  },
  "action": {
    "type": "Enum: create/delete/update",
    "target": "Enum: s3/hottier/local-store/user-list",
    "resource": "Enum: object/user"
  },
  "response": {
    "code": int,
    "body": "String",
    "headers": {
      "key": "value"
    }
  }
}
```